ansible_lab.yml

```diff
@@ -23,65 +22,29 @@
       tags:
         - baseline
 
-    - name: Backup existing SSH config
-      copy: src=/etc/ssh/ssh_config dest=/etc/ssh/ssh_config_backup
-      tags:
-        - ssh
-        - backup_ssh
-        - baseline
-
-    - name: Create group for lab users
+    - name: Create group ansiblelab
       group:
            name=ansiblelab
            state=present
       tags:
         - group
         - baseline
 
-    - name: Create user, home with default password, ssh keys - loop
+    - name: Create users with loop
       user:
           name={{ item }}
-          comment={{ item }}
           password={{ item | password_hash('sha512') }}
           home=/home/{{ item }}
+          createhome=yes
           group=ansiblelab
           generate_ssh_key=yes
-          ssh_key_file=.ssh/id_rsa
-          update_password=always
           state=present
       with_sequence: start=1 end={{ users }} format=ansiblelabuser%d
       tags:
         - users
         - baseline
 
-    - name: unlock sshd_config
```

Comment:
all changes related to sshd should be retained since this playbook is designed to cater to protected (enterprise env's) and non protected environments. Thanks to the idempotency property of ansible, file will not be effected if its not in that sate.